front/components/assistant_builder/types.ts

```diff
@@ -69,13 +69,35 @@ export type AssistantBuilderInitialState = {
   timeFrame: AssistantBuilderState["timeFrame"] | null;
   dustAppConfiguration: AssistantBuilderState["dustAppConfiguration"];
   tablesQueryConfiguration: AssistantBuilderState["tablesQueryConfiguration"];
+  modelConfiguration: AssistantBuilderState["modelConfiguration"];
+  generationConfiguration: AssistantBuilderState["generationConfiguration"];
   handle: string;
   description: string;
   scope: Exclude<AgentConfigurationScope, "global">;
-  instructions: string;
   avatarUrl: string | null;
-  generationSettings: {
-    modelSettings: SupportedModel;
-    temperature: number;
-  } | null;
+  instructions: string;
+};
+
+export const DEFAULT_ASSISTANT_STATE: AssistantBuilderState = {
+  actionMode: "GENERIC",
+  dataSourceConfigurations: {},
+  timeFrame: {
+    value: 1,
+    unit: "month",
+  },
+  dustAppConfiguration: null,
+  tablesQueryConfiguration: {},
+  handle: null,
+  scope: "private",
+  description: null,
+  instructions: null,
+  avatarUrl: null,
+  modelConfiguration: {
+    modelId: GPT_4_TURBO_MODEL_CONFIG.modelId,
```

Comment:
Shouldn't this be conditional based on `isUpgraded`?

Reply:
It was not the case before, I moved this from front/components/assistant_builder/AssistantBuilder.tsx

Reply:
Wait no there's still potential users on FREE_TEST_PLAN good catch 🙏🏻